packages/grid/x-data-grid/src/hooks/features/columns/useGridColumnGrouping.ts

```diff
@@ -0,0 +1,186 @@
+import * as React from 'react';
```

Comment:
Should we move the column grouping feature hooks into their own folder, similar to how the column headers are organized?